proposals/simd/NewOpcodes.md

```diff
@@ -82,7 +82,6 @@
 | -------------------- | ------ | ------------------------ | ------ | ------------------------ | ------ | ------------- | ------ |
 | i8x16.abs            | 0x60   | i16x8.abs                | 0x80   | i32x4.abs                | 0xa0   | ----          | 0xc0   |
 | i8x16.neg            | 0x61   | i16x8.neg                | 0x81   | i32x4.neg                | 0xa1   | i64x2.neg     | 0xc1   |
-| i8x16.any_true       | 0x62   | i16x8.any_true           | 0x82   | i32x4.any_true           | 0xa2   | ----          | 0xc2   |
```

Comment:
For consistency with the rest of this document, perhaps leave in "----" for the operations being removed instead of deleting?

Reply:
Done